File: MPI/mpicheck/tests/functions/4.0/MPI_Type_create_resized_c.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_Datatype var_0;
    MPI_Count var_1;
    MPI_Count var_2;
    MPI_Datatype *var_3;
    int ret;
    /* calls */
    ret = MPI_Type_create_resized_c(var_0, var_1, var_2, var_3);
    ret = PMPI_Type_create_resized_c(var_0, var_1, var_2, var_3);
    return 0;
}

File: MPI/mpicheck/tests/functions/4.0/MPI_Type_create_resized_c.f90

        
        program main
        use mpi
        
        INTEGER var_0
       INTEGER(KIND=MPI_COUNT_KIND) var_1
       INTEGER(KIND=MPI_COUNT_KIND) var_2
       INTEGER var_3
       INTEGER var_4
        call mpi_type_create_resized_c(var_0, var_1, var_2, var_3, var_4)
       call pmpi_type_create_resized_c(var_0, var_1, var_2, var_3, var_4)
        end program main
    